lib/osf-components/addon/components/cedar-metadata-editor/component.ts

```diff
@@ -9,28 +9,50 @@ import CedarMetadataTemplateModel from 'ember-osf-web/models/cedar-metadata-temp
 import RouterService from '@ember/routing/router-service';
 import { task } from 'ember-concurrency';
 import { waitFor } from '@ember/test-waiters';
+import { action } from '@ember/object';
+import Intl from 'ember-intl/services/intl';
+import Toast from 'ember-toastr/services/toast';
+import captureException from 'ember-osf-web/utils/capture-exception';
 
 const { cedarConfig } = config;
 
 interface Args {
     cedarMetadataRecord: CedarMetadataRecordModel;
     cedarMetadataTemplate: CedarMetadataTemplateModel;
     target: AbstractNodeModel | FileModel;
-    edit: boolean;
+    displayArtifactViewer: () => {};
 }
 
 export default class CedarMetadataEditor extends Component<Args> {
+    @service toast!: Toast;
+    @service intl!: Intl;
     @service store!: Store;
     @service router!: RouterService;
 
     cedarConfig = cedarConfig.editorConfig;
+    isEdit = this.args.cedarMetadataRecord ? true : false;
```

Comment:
Looking at the types defined in the `Args` interface, it's implied that this should always be `true`. Could you update the Args interface to `cedarMetadataRecord?: CedarMetadataRecordModel;` to denote that it's an optional arg?